lib/openid_connect/authorization_endpoint.rb

```diff
@@ -0,0 +1,10 @@
+class AuthorizationEndpoint
+  attr_accessor :app, :account, :client, :redirect_uri, :response_type, :scopes, :_request_, :request_uri, :request_object
+  delegate :call, to: :app
+
+  def initialize(allow_approval = false, approved = false)
+    @app = Rack::OAuth2::Server::Authorize.new do |req, res|
```

Comment:
Unused block argument - `res`. If it's necessary, use `_` or `_res` as an argument name to indicate that it won't be used.